mail_attach_existing_attachment/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+##############################################################################
+#
+#     This file is part of mail_attach_existing_attachment,
+#     an Odoo module.
+#
+#     Copyright (c) 2015 ACSONE SA/NV (<http://acsone.eu>)
+#     Copyright (c) 2020 Tecnativa - Manuel Calero
+#
+#     mail_attach_existing_attachment is free software:
+#     you can redistribute it and/or modify it under the terms of the GNU
+#     Affero General Public License as published by the Free Software
+#     Foundation,either version 3 of the License, or (at your option) any
+#     later version.
+#
+#     mail_attach_existing_attachment is distributed
+#     in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with mail_attach_existing_attachment.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```

Comment:
We can take advantage of this migration to change the header of this file according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L1-L2